client/less/main.less

```diff
@@ -510,6 +510,10 @@ form.update-email .btn{
   padding-left: 50px;
 }
 
+.challenges-instructions-panel {
```

Comment:
I think this would be better in `./common/app/routes/Challenges/challenges.less` using the `@{ns}` namespace like the other class declarations in that file.